common/helpdata.cpp

```diff
@@ -3804,28 +3780,20 @@ void helptext_extra(char *buf, size_t bufsz, struct player *pplayer,
         CATLSTR(
             buf, bufsz,
             _("\nTime to build and output bonus depends on terrain:\n\n"));
-        CATLSTR(buf, bufsz,
-                /* TRANS: Header for fixed-width road properties table.
-                 * TRANS: Translators cannot change column widths :( */
-                _("Terrain       Time     Bonus F/P/T\n"
-                  "----------------------------------\n"));
+        cat_snprintf(buf, bufsz, "| %s | %s | %s |\n", _("Terrain"),
+                     _("Time"), _("Bonus F/P/T"));
+        CATLSTR(buf, bufsz, "| :-- | --: | :-- |\n");
```

Comment:
Suggest aligning the last column to the right. Appears in Terrain > Terrain Alterations > Road